### DISCUSSION TITLE
文章随机图怎么实现的呢？

### DISCUSSION THREAD

**Discussion by 503611908** [mirai-mamori/Sakurairo#517] [Q&A]
你好，请问你的文章随机图是怎么实现的呢？如何实现文章特色图片调用随机图api呢？

**Comment by Davy-Chendy** [comment]
在inc/theme_plus.php（大概290行）找到

$full_image_url = !empty($full_image_url) ? $full_image_url[0] : null;

改为

$full_image_url = !empty($full_image_url) ? $full_image_url[0] :get_random_bg_url();
